docs/card/card-quiet-small.yml

```diff
@@ -42,3 +42,46 @@ markup: |
       </div>
     </div>
   </div>
+  
+  <h2>Focused Card item within a grid</h3>
+  <div role="grid">
+    <div style="width: 112px; height: 136px; outline: 0;" role="row" tabindex="0" class="focus-ring">
+      <div class="spectrum-Card spectrum-Card--quiet spectrum-Card--small is-focused" role="gridcell">
```

Comment:
Ok so this is interesting -- you're simulating `:focus-ring` in this example, but also applying `.is-focused` to the Card under it... Doesn't it seem like `.is-focused` should be the only thing required to make it look focused?